code/game/machinery/limbgrower.dm

```diff
@@ -180,7 +199,7 @@
 			for(var/reagent_id in consumed_reagents_list)
 				consumed_reagents_list[reagent_id] *= production_coefficient
 				if(!reagents.has_reagent(reagent_id, consumed_reagents_list[reagent_id]))
-					audible_message("<span class='notice'>\The [src] buzzes.</span>")
+					audible_message("<span class='warning'>\The [src] buzzes, with a screen showing: INSUFFICENT REAGENTS</span>")
```

Comment:
```suggestion
					audible_message("<span class='warning'>\The [src] buzzes and states "INSUFFICENT REAGENTS"</span>")
```
(audible_message shows to blind characters and them being able to read the display is a bit weird)